velox/type/tests/VariantTest.cpp

```diff
@@ -285,3 +286,66 @@ TEST(VariantTest, opaqueSerializationNotRegistered) {
       std::make_shared<opaqueSerializationTestStruct>());
   EXPECT_THROW(opaqueAfterRegistration.serialize(), VeloxException);
 }
+
+TEST(VariantTest, toJson) {
+  auto rowTypePtr = ROW({{"c0", DECIMAL(20, 3)}});
+  EXPECT_EQ(
+      "[123456.789]",
+      variant::row({static_cast<int128_t>(123456789)}).toJson(rowTypePtr));
+  rowTypePtr = ROW({{"c0", DECIMAL(10, 2)}});
```

Comment:
consider adding a new line before this to improve readability. Same for below before each new type.

Reply:
Fixed. Added a new line before each case.